### PR TITLE
feat: Card Styling for announcements

### PR DESCRIPTION
**Pull request by Golodhros**
### Summary of Changes
Adds styling to the cards we'll use on announcements

Amundsen_-_Data_Discovery_Portal

### Tests
Adds new tests

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes, including screenshots of any UI changes. 
- [ ] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [ ] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)

### CODE REVIEW

**Comment by allisonsuarez** [comment]
We have simmer loading components in a few places, granted all of them look a bit different depending on what component they are used for, but I wonder if we can hav them be together in some way? Other than that thought looks good to me.

**Comment by Golodhros** [comment]
> We have simmer loading components in a few places, granted all of them look a bit different depending on what component they are used for, but I wonder if we can have them be together in some way? Other than that thought looks good to me.

Good call! Up to now, we have only reused the animation style, as the loaders have been pretty different. I think we are now in the position to abstract it a bit more, as Knowl wants us to use only two sizes of lines.

File: amundsen_application/static/js/components/HomePage/index.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { resetSearchState } from 'ducks/search/reducer';
 import { UpdateSearchStateReset } from 'ducks/search/types';
 import SearchBar from 'components/common/SearchBar';
 import TagsList from 'components/common/TagsList';
+import Card from 'components/common/Card';
```
Comment (ttannis) [inline amundsen_application/static/js/components/HomePage/index.tsx:19]: assume this is just for testing.

File: amundsen_application/static/js/components/common/Card/index.tsx [hunk 2]
```diff
@@ -1,35 +1,41 @@
 import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import './styles.scss';
 
 export interface CardProps {
-  title: string;
+  title?: string;
```
Comment (ttannis) [inline amundsen_application/static/js/components/common/Card/index.tsx:7]: nit: Just wondering why all properties are optional, to be more generic/reuseable?

File: amundsen_application/static/js/components/HomePage/index.tsx (hunk 1 above)
Comment (Golodhros) [reply amundsen_application/static/js/components/HomePage/index.tsx:19]: Yeah, will clean when done!

File: amundsen_application/static/js/components/common/Card/index.tsx (hunk 2 above)
Comment (Golodhros) [reply amundsen_application/static/js/components/common/Card/index.tsx:7]: It is a byproduct of integrating the loading state inside the card, as we won't have any of the properties available when loading them.

**Pull request merged**